blocks/flip-cards/flip-cards.js

```diff
@@ -1,16 +1,67 @@
-export default function decorate(block) {
+async function getHtml(url) {
+  let html;
+  const page = await fetch(url);
+  if (page.ok) {
+    html = await page.text();
+  } else {
+    html = '';
+  }
+  const retEl = document.createElement('div');
+  retEl.innerHTML = html;
+  return retEl;
+}
+
+function getDocumentMetadata(name, document) {
+  const attr = name && name.includes(':') ? 'property' : 'name';
+  const meta = [...document.querySelectorAll(`meta[${attr}="${name}"]`)]
+    .map((m) => m.content)
+    .join(', ');
+  return meta || '';
+}
+
+export default async function decorate(block) {
   const cols = [...block.firstElementChild.children];
+  const rows = [...block.children];
+  const fronts = [];
+  const backs = [];
+  const links = [];
```

Comment:
Are you 100% sure there are no other pages that require the manual authoring of the flip-cards? Are they fetched from subpages everywhere? Otherwise we should keep both ways of populating them and should just add an additional style that triggers the auto-fetch stuff.

Reply:
Hey; yes; they should be! But I added the old logic just in case the feature is necessary.